packages/suite/src/actions/wallet/coinjoinAccountActions.ts

```diff
@@ -547,7 +537,6 @@ export const createCoinjoinAccount =
                 },
             ),
         );
-        dispatch(coinjoinAccountCreate(account.payload));
 
         console.log(`CoinjoinAccount created: ${getAccountProgressHandle(account.payload)}`);
```

Comment:
forgotten log?

Reply:
Not really, there's more of them now, we've used it together with `--log-ui` flag to debug discovery process. Probably the only alternative to using the console log was to implement dedicated desktop API method for propagating logs to the log file, which seems too heavyweight.